src/Symfony/Bridge/Twig/Extension/IntlExtension.php

```diff
@@ -0,0 +1,411 @@
+        'after_prefix' => NumberFormatter::PAD_AFTER_PREFIX,
+        'before_suffix' => NumberFormatter::PAD_BEFORE_SUFFIX,
+        'after_suffix' => NumberFormatter::PAD_AFTER_SUFFIX,
+    );
+    private $numberTextAttributes = array(
+        'positive_prefix' => NumberFormatter::POSITIVE_PREFIX,
+        'positive_suffix' => NumberFormatter::POSITIVE_SUFFIX,
+        'negative_prefix' => NumberFormatter::NEGATIVE_PREFIX,
+        'negative_suffix' => NumberFormatter::NEGATIVE_SUFFIX,
+        'padding_character' => NumberFormatter::PADDING_CHARACTER,
+        'currency_mode' => NumberFormatter::CURRENCY_CODE,
+        'default_ruleset' => NumberFormatter::DEFAULT_RULESET,
+        'public_rulesets' => NumberFormatter::PUBLIC_RULESETS,
+    );
+    private $symbols = array(
+        'decimal_separator' => NumberFormatter::DECIMAL_SEPARATOR_SYMBOL,
+        'grouping_separator' => NumberFormatter::GROUPING_SEPARATOR_SYMBOL,
+        'pattern_separator' => NumberFormatter::PATTERN_SEPARATOR_SYMBOL,
+        'percent' => NumberFormatter::PERCENT_SYMBOL,
+        'zero_digit' => NumberFormatter::ZERO_DIGIT_SYMBOL,
+        'digit' => NumberFormatter::DIGIT_SYMBOL,
+        'minus_sign' => NumberFormatter::MINUS_SIGN_SYMBOL,
+        'plus_sign' => NumberFormatter::PLUS_SIGN_SYMBOL,
+        'currency' => NumberFormatter::CURRENCY_SYMBOL,
+        'intl_currency' => NumberFormatter::INTL_CURRENCY_SYMBOL,
+        'monetary_separator' => NumberFormatter::MONETARY_SEPARATOR_SYMBOL,
+        'exponential' => NumberFormatter::EXPONENTIAL_SYMBOL,
+        'permill' => NumberFormatter::PERMILL_SYMBOL,
+        'pad_escape' => NumberFormatter::PAD_ESCAPE_SYMBOL,
+        'infinity' => NumberFormatter::INFINITY_SYMBOL,
+        'nan' => NumberFormatter::NAN_SYMBOL,
+        'significant_digit' => NumberFormatter::SIGNIFICANT_DIGIT_SYMBOL,
+        'monetary_grouping_separator' => NumberFormatter::MONETARY_GROUPING_SEPARATOR_SYMBOL,
+    );
+
+    public function __construct($defaultLocale = null)
+    {
+        $this->defaultLocale = $defaultLocale;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getFilters()
+    {
+        return array(
+            new \Twig_SimpleFilter('localized_country', array($this, 'getLocalizedCountry')),
+            new \Twig_SimpleFilter('localized_currency', array($this, 'getLocalizedCurrency')),
+            new \Twig_SimpleFilter('localized_currency_name', array($this, 'getLocalizedCurrencyName')),
+            new \Twig_SimpleFilter('currency_symbol', array($this, 'getCurrencySymbol')),
+            new \Twig_SimpleFilter('localized_date', array($this, 'getLocalizedDate')),
+            new \Twig_SimpleFilter('localized_language', array($this, 'getLocalizedLanguage')),
+            new \Twig_SimpleFilter('localized_locale', array($this, 'getLocalizedLocale')),
+            new \Twig_SimpleFilter('localized_number', array($this, 'getLocalizedNumber')),
```

Comment:
IMO, all filters depending only on ext-intl features should not be duplicated from ``Twig_Extensions_Extension_Intl``. There is no benefit about maintaining it twice, and there is no benefit about forcing people to install symfony/intl if they only need to format a number rather than getting the list of countries